### PR TITLE
MinimumCostPolygonTriangulation

### PR DESCRIPTION
**Pull request by bbowles98**
Issue #87

### CODE REVIEW

File: src/main/java/com/williamfiset/algorithms/geometry/MinimumCostPolygonTriangulation.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ * <p>Time Complexity: O(n^3)
+ *
+ * @author Bryan Bowles
+ */
+package com.williamfiset.algorithms.geometry;
+
+import java.awt.geom.Point2D;
+
+// Problem explanation: https://www.geeksforgeeks.org/minimum-cost-polygon-triangulation/
+public class MinimumCostPolygonTriangulation {
```
Comment (williamfiset) [inline src/main/java/com/williamfiset/algorithms/geometry/MinimumCostPolygonTriangulation.java:13]: This code makes an assumption that the polygon is Convex and that the points are in CW and CCW order, correct? Can you either rename the class, add a comment, or both to reflect that?

File: src/test/java/com/williamfiset/algorithms/geometry/MinimumCostPolygonTriangulationTest.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    Point2D[] pts = new Point2D[5];
+
+    pts[0] = new Point2D.Double(0, 0);
+    pts[1] = new Point2D.Double(1, 0);
+    pts[2] = new Point2D.Double(2, 1);
+    pts[3] = new Point2D.Double(1, 2);
+    pts[4] = new Point2D.Double(0, 2);
+
+    double cost = MinimumCostPolygonTriangulation.minimumCostTriangulation(pts);
+    assert (Math.abs(cost - 15.3) < .001);
```
Comment (williamfiset) [inline src/test/java/com/williamfiset/algorithms/geometry/MinimumCostPolygonTriangulationTest.java:19]: Do you want to try using assertThat from the Google Truth framework to check relative tolerance? I would also define the tolerance as a static final constant in this file.

import static com.google.common.truth.Truth.assertThat;
https://truth.dev/floating_point.html

So, something like:
assertThat(value).usingTolerance(1.0e-10).isEqualTo(otherValue);

File: src/test/java/com/williamfiset/algorithms/geometry/MinimumCostPolygonTriangulationTest.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
+package com.williamfiset.algorithms.geometry;
+
+import java.awt.geom.*;
+import org.junit.*;
+
+public class MinimumCostPolygonTriangulationTest {
```
Comment (williamfiset) [inline src/test/java/com/williamfiset/algorithms/geometry/MinimumCostPolygonTriangulationTest.java:6]: Thanks for adding tests!

File: src/main/java/com/williamfiset/algorithms/geometry/MinimumCostPolygonTriangulation.java (hunk 1 above)
Comment (bbowles98) [reply src/main/java/com/williamfiset/algorithms/geometry/MinimumCostPolygonTriangulation.java]: Good catch! updated

**Comment by williamfiset** [comment]
Bryan, thanks for the PR

**Pull request merged**